cookbooks/.gitignore

```diff
@@ -1,3 +1,6 @@
 /apt
 /build-essential
 /cdo-aws
+.kitchen
+*/Berksfile.lock
```

Comment:
I was trying to see if it would be possible for us to depend only on `cookbooks/Berksfile[.lock]` for all of our cookbooks without having to create separate `Berksfile[.lock]` files in each cookbook's folder, but it doesn't seem to be possible/recommended. I'll add in the extra files and remove this ignore line.